common/src/main/webapp/less/style.less

```diff
@@ -2035,6 +2037,12 @@ html {
   }
 }
 
+.tile-xs-width {
+  .explore-shared-edit-warning {
+    display: none;
```

Comment:
No warning for very narrow displays?

Reply:
So, and this maybe a bad take. We are hiding the editors for narrow screens thus the warning isnt necessarily. If we were going to show the editors we should certainly restore the warning

Reply:
That probably makes sense. It would be pretty hard to edit a target on, say, a phone.